sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRDD.scala

```diff
@@ -390,7 +390,9 @@ private[sql] class JDBCRDD(
     val sqlText = s"SELECT $columnList FROM $fqTable $myWhereClause"
     val stmt = conn.prepareStatement(sqlText,
         ResultSet.TYPE_FORWARD_ONLY, ResultSet.CONCUR_READ_ONLY)
-    val fetchSize = properties.getProperty("fetchsize", "0").toInt
+    val fetchSize = properties.getProperty(JdbcUtils.JDBC_BATCH_FETCH_SIZE, "0").toInt
+    require(fetchSize >= 0,
+      s"Invalid value `${fetchSize.toString}` for parameter `${JdbcUtils.JDBC_BATCH_FETCH_SIZE}`.")
```

Comment:
https://github.com/gatorsmile/spark/blob/b999b8a1474fc9d4f8d3a4c94694fbc40572111a/sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala#L181-L183 

https://github.com/gatorsmile/spark/blob/b999b8a1474fc9d4f8d3a4c94694fbc40572111a/sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala#L394-L395

Above are the document changes. Please check if they are ok. Thanks!